src/CollapsibleItem.js

```diff
@@ -67,24 +82,27 @@ class CollapsibleItem extends Component {
 }
 
 CollapsibleItem.propTypes = {
-  children: PropTypes.node,
   header: PropTypes.string.isRequired,
   icon: PropTypes.string,
   onSelect: PropTypes.func,
   /**
-   * If the item is expanded by default
+   * If the item is expanded by default. Overridden if the parent Collapsible is an accordion.
    * @default false
    */
   expanded: PropTypes.bool,
   /**
-   * The value to pass to the onSelect callback
+   * The value to pass to the onSelect callback.
    */
   eventKey: PropTypes.any,
   /**
    * The node type of the header
    * @default a
    */
-  node: PropTypes.node
+  node: PropTypes.node,
+  /**
+   * The scroll behavior for scrollIntoView
+   */
+  scroll: PropTypes.string
```

Comment:
The oneOf would be perfect, thanks for sharing. I'll add that. We wouldn't want to put this in the collapsible component because in my testing the scroll would fire before the collapsibleItem re-render and it wouldn't work as expected.